tests/php/classes/test-class-wc-rest-connect-packages-controller.php

```diff
@@ -20,7 +20,7 @@ class WP_Test_WC_REST_Connect_Packages_Controller extends WC_REST_Unit_Test_Case
 	/**
 	 * @inherit
 	 */
-	public static function setupBeforeClass() {
+	public static function set_up_before_class() {
```

Comment:
As per changes in WP tests: https://make.wordpress.org/core/2021/09/27/changes-to-the-wordpress-core-php-test-suite/